wetmap/src/components/googleMap/index.tsx

```diff
@@ -233,7 +233,7 @@ export default function MapLoader() {
 
   const handleDragEnd = () => {
     if (pinRef) {
-      const position = pinRef;
+      const position = pinRef.position;
```

Comment:
@Freem11 Could you please take a look at this. Just to be sure this is the right way to get "position"

Reply:
I have fixed it, I also cleaned up the site submitter view and index of typescript errors an unused items while I was at it, there was an error that was popping up because <text></text> was being used